src/components/base/button/styles.module.scss

```diff
@@ -15,3 +15,9 @@
 .primary {
   background-color: $primary-color;
 }
+
+.console {
```

Comment:
can you change the type name to another one more generic? maybe menu-button or something like that